js_modules/dagit/packages/core/src/overview/OverviewAssetsRoot.tsx

```diff
@@ -0,0 +1,433 @@
+                <Box flex={{direction: 'row', alignItems: 'center', gap: 6}}>
+                  <div
+                    style={{
+                      width: '12px',
+                      height: '12px',
+                      border: `2px solid ${Colors.Gray500}`,
+                      borderRadius: '50%',
+                    }}
+                  />
+                  {statuses.missing.length}
+                </Box>
+              </Tag>
+            </SelectOnHover>
+          ) : (
+            0
+          )}
+        </Cell>
+        <Cell isLoading={!!statuses.loading}>
+          {statuses.failed.length ? (
+            <SelectOnHover assets={statuses.failed}>
+              <Tag intent="danger">
+                <Box flex={{direction: 'row', alignItems: 'center', gap: 6}}>
+                  <div
+                    style={{
+                      width: 0,
+                      height: 0,
+                      borderLeft: '6px solid transparent',
+                      borderRight: '6px solid transparent',
+                      borderBottom: '10px solid red',
+                      display: 'inline-block',
+                    }}
+                  />
+                  {statuses.failed.length}
+                </Box>
+              </Tag>
+            </SelectOnHover>
+          ) : (
+            0
+          )}
+        </Cell>
+        <Cell isLoading={!!statuses.loading}>
+          {statuses.inprogress.length ? (
+            <SelectOnHover assets={statuses.inprogress}>
+              <Tag intent="primary" icon="spinner">
+                {statuses.inprogress.length}
+              </Tag>
+            </SelectOnHover>
+          ) : (
+            0
+          )}
+        </Cell>
+        <Cell isLoading={!!statuses.loading}>
+          {statuses.successful.length ? (
+            <SelectOnHover assets={statuses.successful}>
+              <Tag intent="success">
+                <Box flex={{direction: 'row', alignItems: 'center', gap: 6}}>
+                  <div
+                    style={{
+                      backgroundColor: Colors.Green500,
+                      width: '10px',
+                      height: '10px',
+                      borderRadius: '50%',
+                    }}
+                  />
+                  {statuses.successful.length}
+                </Box>
+              </Tag>
+            </SelectOnHover>
+          ) : (
+            0
+          )}
+        </Cell>
+      </RowGrid>
+    </Row>
+  );
+}
+
+const RowGrid = styled(Box)`
+  display: grid;
+  grid-template-columns: ${TEMPLATE_COLUMNS};
+  height: 100%;
+  > * {
+    padding-top: 26px 0px;
+  }
+`;
+
+const Cell = ({children, isLoading}: {children: React.ReactNode; isLoading?: boolean}) => {
+  return (
+    <RowCell style={{color: Colors.Gray900}}>
+      {isLoading ? (
+        <Box flex={{justifyContent: 'center', alignItems: 'center'}} style={{height: '82px'}}>
+          <Spinner purpose="body-text" />
```

Comment:
If I am reading this correctly, the loading state of the table will end up with a ton of spinners, which might be a bit jarring.

Reply:
yeah its from the figma.